admin/js/wizard.js

```diff
@@ -19,6 +19,62 @@ function buildFormData (formData, data, parentKey) {
     formData.append(parentKey, value);
   }
 };
+function installPlugin(event) {
+  if (event){
+    event.preventDefault();
+  }
+  var formData = new FormData(event.target, event.submitter);
+  var button = formData.get('button');
+
+  if (button !== 'all'){
+    var nonce = formData.get(button)
+    installSlug(button, nonce);
+  } else {
+    try {
+      console.log(formData.entries());
+      formData.entries()
+      .filter((entry) => entry[0] !== 'button')
+      .forEach((plugin) => installSlug(plugin[0], plugin[1]));
+    } catch (error) {
+      console.error(error);
+      showMessage('Error setting option', 'error');
+    }
+  }
+}
+function installSlug(plugin, nonce) {
+  try {
+    plugin = JSON.parse(plugin);
+  } catch (error) {
+    showMessage(plugin, 'error')
+  }
+  console.log('url: ' + plugin['url']);
+  console.log('slug: ' + plugin['slug']);
+  if (plugin['url'] == null) {
```

Comment:
Can we do this logic in the js click event function above? If this function is really just installing by slug, you should just pass the slug as the parameter and check that it exists before you call it. Right now, you could sequentially call all 3 install plugins if it's a URL when you really just need to go from the click event to the `installByUrl` if you are given a URL.